src/WebJobs.Extensions.DurableTask/Listener/DurableTaskScaleMonitor.cs

```diff
@@ -86,7 +87,7 @@ public async Task<DurableTaskTriggerMetrics> GetMetricsAsync()
             }
             catch (StorageException e)
             {
-                this.traceHelper.ExtensionWarningEvent(this.hubName, this.functionName.Name, string.Empty, e.ToString());
+                this.logger.LogWarning(e.ToString(), this.hubName, this.functionName);

```

Comment:
What's this log message expected to look like? I thought `LogWarning` was supposed to take a template as the first parameter and arguments for that template as the subsequent parameters.

Reply:
`LogWarning` is supposed to take a template and then parameters. I updated it to that format. Thanks for pointing that out!